src/cache_cf.cc

```diff
@@ -4891,12 +4953,46 @@ static void free_ftp_epsv(acl_access **ftp_epsv)
     FtpEspvDeprecated = false;
 }
 
+/// Like parseTimeLine() but does not require the timeunit to be specified.
+/// If missed, the default 'second' timeunit is assumed.
+static std::chrono::seconds
+ParseUrlRewriteTimeout()
+{
+    const auto valueToken = ConfigParser::NextToken();
+    if (!valueToken)
+        self_destruct();
+
+    using Seconds = std::chrono::seconds;
+
+    const auto parsedValue = xatof(valueToken);
+
+    std::chrono::nanoseconds parsedUnitDuration;
+
+    const auto token = parsedValue ? ConfigParser::PeekAtToken() : nullptr;
+
+    if (parsedValue) {
+        if (parseTimeUnit<Seconds>(token, parsedUnitDuration))
+            (void)ConfigParser::NextToken();
+        else {
+            const auto defaultParsed = parseTimeUnit<Seconds>(T_SECOND_STR, parsedUnitDuration);
+            assert(defaultParsed);
+            debugs(3, DBG_CRITICAL, "WARNING: " << ConfigPosition() << "missing time unit, using deprecated default '" << T_SECOND_STR << "'");
+        }
+    }
+
+    CheckTimeValue(parsedValue, parsedUnitDuration);
```

Comment:
Please check whether this should return nanoseconds. All callers seem to want that.

Reply:
Done.